plugins/example/example.coffee

```diff
@@ -0,0 +1,22 @@
+(() ->
+  class HelloWorldPlugin
```

Comment:
Possibly, but any plugin class should not be a nice extra to make things easier and not mandatory. I didn't know what would go in it currently, so I left it out--everything I think of would just go in the API.

(Optional especially because people may want to write code in pure JS, which AFIK can't interoperate with coffeescript classes)